MODULE.bazel

```diff
@@ -19,6 +19,7 @@ use_repo(
     "com_github_jetbrains_kotlin",
     "com_github_jetbrains_kotlin_git",
     "com_github_pinterest_ktlint",
+    "java_stub_template_txt",
```

Comment:
This is being removed from Bazel and doesn't seem to be exposed by rules_java anymore.